geomesa-core/src/test/scala/org/locationtech/geomesa/core/iterators/AttributeIndexIteratorTest.scala

```diff
@@ -115,12 +119,10 @@ class AttributeIndexIteratorTest extends Specification with TestWithDataStore {
         "name = 'b'",
         "name < 'b'",
         "name > 'b'",
-        "name is NULL",
         "dtg TEQUALS 2014-01-01T12:30:00.000Z",
         "dtg = '2014-01-01T12:30:00.000Z'",
         "dtg BETWEEN '2012-01-01T12:00:00.000Z' AND '2013-01-01T12:00:00.000Z'",
-        "age < 10",
-        "name = 'b' AND BBOX(geom, 30, 30, 50, 50)"
```

Comment:
We prioritize spatial filters now (unless the attribute has been marked high-cardinality). Using the order was problematic through geoserver, since geoserver doesn't respect the order of your cql when passing it along.